ultra/ultra/baselines/common/social_vehicle_config.py

```diff
@@ -19,23 +19,14 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
-from ultra.baselines.common.social_vehicle_extraction import *
 from ultra.baselines.common.social_vehicles_encoders.pointnet_encoder import PNEncoder
 from ultra.baselines.common.social_vehicles_encoders.pointnet_encoder_batched import (
     PNEncoderBatched,
 )
-from ultra.baselines.common.social_vehicle_extraction import *
 from ultra.baselines.common.social_vehicles_encoders.precog_encoder import (
     PrecogFeatureExtractor,
 )
 
-social_vehicle_extractors = {
-    "pointnet_encoder": extract_social_vehicle_state_pointnet,
-    "pointnet_encoder_batched": extract_social_vehicle_state_pointnet,
-    "precog_encoder": extract_social_vehicle_state_default,
-    "no_encoder": extract_social_vehicle_state_default,
-}
-
```

Comment:
We no longer use specific extraction functions. Perhaps we should `social_vehicle_extraction.py` however, because we could use those functions in another type of observation adapter.